app/controllers/prison/visits_controller.rb

```diff
@@ -65,6 +61,14 @@ def cancel
 
 private
 
+  def visit_is_processable
+    visit = load_visit
```

Comment:
it would be nice to avoid loading the visit twice per action (even if the second time it is memoized)